src/tf_container/__init__.py

```diff
@@ -11,8 +11,8 @@
 #  express or implied. See the License for the specific language governing 
 #  permissions and limitations under the License.
 
-from tf_container.train import train
+from tf_container.train_entry_point import train
```

Comment:
Where is train_entry_point coming from?

Reply:
This change renames train.py to train_entry_point.py, since otherwise there is no way to access the module tf_container.train. __init__.py imports the train function, which overrides the train module.

This affects our ability to test other functions from train_entry_point.py.